content/en/docs/chart_template_guide/control_structures.md

```diff
@@ -259,11 +259,13 @@ The next control structure to look at is the `with` action. This controls
 variable scoping. Recall that `.` is a reference to _the current scope_. So
 `.Values` tells the template to find the `Values` object in the current scope.
 
-The syntax for `with` is similar to a simple `if` statement:
+The syntax for `with` is similar to the `if`/`else` block:
 
 ```
 {{ with PIPELINE }}
-  # restricted scope
+  # restricted scope, if PIPELINE is true
+{{ else }}
+  # restricted scope, if PIPELINE is false
```

Comment:
```
{{with pipeline}} T1 {{else}} T0 {{end}}
	If the value of the pipeline is empty, dot is unaffected and T0
	is executed; otherwise, dot is set to the value of the pipeline
	and T1 is executed.
```

Reading the official docs, the else scope is not restricted.

Reply:
Oh, that's so obvious when you say it, thanks for catching it! 🤦 Fix incoming!

Reply:
I've pushed a new commit now with both proper signoff and changed this to not be restriceted. Mark this as resolved if you are happy with the change. Otherwise, I'll change it again according to your suggestions.